src/IssueViz.Security/Hotspots/HotspotReviewPriorityProvider.cs

```diff
@@ -1,223 +0,0 @@
-﻿/*
- * SonarLint for Visual Studio
- * Copyright (C) 2016-2024 SonarSource SA
- * mailto:info AT sonarsource DOT com
- *
- * This program is free software; you can redistribute it and/or
- * modify it under the terms of the GNU Lesser General Public
- * License as published by the Free Software Foundation; either
- * version 3 of the License, or (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- * Lesser General Public License for more details.
- *
- * You should have received a copy of the GNU Lesser General Public License
- * along with this program; if not, write to the Free Software Foundation,
- * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
- */
-
-using System;
-using System.Collections.Generic;
-using System.ComponentModel.Composition;
-using SonarLint.VisualStudio.IssueVisualization.Security.Hotspots.Models;
-
-namespace SonarLint.VisualStudio.IssueVisualization.Security.Hotspots
-{
-    /// <summary>
-    /// Provides a mapping for hotspots to their review priority
-    /// </summary>
-    public interface IHotspotReviewPriorityProvider
-    {
-        /// <summary>
-        /// Supported languages:
-        /// <list type="bullet">
-        /// <item><description>JavaScript</description></item>
-        /// <item><description>TypeScript</description></item>
-        /// <item><description>C++</description></item>
-        /// <item><description>C</description></item>
-        /// </list>
-        /// </summary>
-        /// <param name="hotspotKey">Hotspot Rule Key</param>
-        /// <returns>Associated review priority or null if not mapped</returns>
-        HotspotPriority? GetPriority(string hotspotKey);
-    }
-
-    [Export(typeof(IHotspotReviewPriorityProvider))]
-    [PartCreationPolicy(CreationPolicy.Shared)]
-    public class HotspotReviewPriorityProvider : IHotspotReviewPriorityProvider
-    {
-        // In order to save time and considering the fact that hotspots are no longer being updated, we decided to make a hardcoded list of priorities by rule key
-        // See (https://github.com/SonarSource/sonarlint-visualstudio/issues/4532)
-        private static readonly Dictionary<string, HotspotPriority> KeyToPriorityMap = new Dictionary<string, HotspotPriority>
-        {
-            { "cpp:S5443", HotspotPriority.Low },
```

Comment:
We've not migrated CFamily hotspots to slcore yet, so this shouldn't have been deleted for those.

Reply:
I am not sure I understand this comment. The CFamily assembly is not referencing at all this class. So it should not be impacted.

Reply:
Yes but how do CFamily hotspots get this priority if the class is deleted? They are using the same old model without the priority